nano/lib/stats.cpp

```diff
@@ -545,6 +545,173 @@ std::string nano::stat::detail_to_string (uint32_t key)
 	return res;
 }
 
+std::string nano::stat::detail_raw_to_string (nano::stat::detail detail)
```

Comment:
This duplicates everything in `detail_to_string`. Maybe convert detail to key and forward to `detail_to_string`.